### PR TITLE
Add Bigfoot workshop

### PR DESCRIPTION
**Pull request by jkwok91**
Polishing @ad510's Bigfoot workshop in https://github.com/hackclub/hackclub/pull/669 and adding more changes.
Bigfoot workshop has been added to master list of workshops in workshops/README.md

This closes https://github.com/hackclub/hackclub/pull/669.

### CODE REVIEW

File: workshops/find_bigfoot/README.md [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+## Set up the project
+
+If you've done other Hack Club workshops before, this should be pretty
+straightforward.
+
+### Create a blank HTML file
+
+1. Go to https://c9.io
+2. Under the workspace called `USERNAME.github.io` where `USERNAME` is your
+GitHub username, click **Open** to open the workspace.
```
Comment (zachlatta) [inline workshops/find_bigfoot/README.md]: Should be indented by three spaces.

File: workshops/find_bigfoot/README.md [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+3. Make a new folder called `find_bigfoot`.
+4. In the `find_bigfoot` folder, make a new file called `index.html`.
+5. Double-click `index.html` to open the file.
+
+### Add standard HTML template
```
Comment (zachlatta) [inline workshops/find_bigfoot/README.md]: Titlecase!

**Comment by zachlatta** [comment]
Left a couple nitpicky comments, but will just fix them in a separate pull request after I merge.

Thanks! And thanks @ad510 for building this :-). Looks great.

**Comment by ad510** [comment]
Yay, it's merged now! I have some ideas on how to improve this further and I want to follow it up with more workshops, but I'm going to be very busy with schoolwork this week. Next week may be a little better.

Sorry to hear about @jonleung leaving. Have you been able to fill in for the things he was best at?

**Comment by zachlatta** [comment]
Sounds great :-). Good luck with all the school work!

So far we've been able to pick up the workload – @jkwok91 is on a full-time
contract this week to help with that. Definitely wouldn't be possible
without her or @paked, who has also stepped up (among others in the
community).

On Tue, Feb 16, 2016 at 9:40 PM, Andrew Downing <notifications@github.com>
wrote:

> Yay, it's merged now! I have some ideas on how to improve this further and
> I want to follow it up with more workshops, but I'm going to be very busy
> with schoolwork this week. Next week may be a little better.
>
> Sorry to hear about @jonleung <https://github.com/jonleung> leaving. Have
> you been able to fill in for the things he was best at?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/hackclub/pull/724#issuecomment-185036376>.
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>

**Pull request merged**